src/generator/pollers.ts

```diff
@@ -26,26 +26,52 @@ export async function generatePollers(session: Session<CodeModel>): Promise<stri
   imports.add('time');
   imports.add('errors');
   imports.add('encoding/json');
-  text += imports.text();
-
+  let text = '';
+  let pageableLRO = false;
   const pollers = <Array<PollerInfo>>session.model.language.go!.pollerTypes;
   pollers.sort((a: PollerInfo, b: PollerInfo) => { return sortAscending(a.name, b.name) });
   for (const poller of values(pollers)) {
     const pollerInterface = pascalCase(poller.name);
     let responseType = '';
     let rawResponse = ''; // used to access the raw response field on response envelopes
     if (poller.schema === undefined) {
-      responseType = 'http.Response';
+      responseType = '*http.Response';
     } else {
-      responseType = poller.schema.language.go!.responseType.name;
+      if (poller.op.language.go!.pageableType) {
```

Comment:
This `if/else` block can be condensed into the block above.

Reply:
Done